ts/lib/auth/googleauth.ts

```diff
@@ -411,7 +423,8 @@ export default class GoogleAuth {
    * @param {string} - The API key string
    * @param {function=} - Optional callback function
    */
-  public fromAPIKey(apiKey, opt_callback) {
+  public fromAPIKey(
+      apiKey: string, opt_callback: (err: Error, client?: JWTClient) => void) {
```

Comment:
nit: `opt_callback` seems wrong as-is. The `opt` implies that this should be `opt_callback?`, but I would recommend changing it to `callback?`.